history.md

```diff
@@ -0,0 +1,67 @@
+# 0.0.1
+- Support singleton client configuration
```

Comment:
Capitalize the first letter
of 'singleton'?

Reply:
will fix it in next merge